### ISSUE TITLE
[Nightly] Crash on adding photo via camera to a cache

### ISSUE THREAD

**Issue by murggel** [cgeo/cgeo#13682]
### Describe your problem!

Crash, when I want to add a photo to a cache via camera in image-gallery 

### How to reproduce?

Open a cache
Go to images
Add photo via camera-symbol

### Actual result after these steps?

The cgeo crash and no photo is saved 

### Expected result after these steps?

Photo is added to own images

### Reproducible

Yes

### c:geo Version

2022.11.16-NB-c66e387

### System information

```text
## System information

c:geo version: 2022.11.16-NB-c66e387

Device:
-------
- Device type: SM-G980F (x1sxx, samsung)
- Available processors: 8
- Android version: 11
- Android build: RP1A.200720.012.G980FXXSCDUJ5
- Screen resolution: 1080x2178px (360x726dp)
- Pixel density: 3.3000002
- System font scale: 1.1 / used scale: 1.1
- Sailfish OS detected: false
- Google Play services: enabled - 22.44.17 (150408-488410709)

Sensor and location:
-------
- Low power mode: active
- Compass capabilities: yes
- Rotation vector sensor: present
- Orientation sensor: present
- Magnetometer & Accelerometer sensor: present
- Direction sensor used: rotation vector

Program settings:
-------
- Settings: v9, Count:180
- Set language: de
- System date format: dd.MM.yy
- Time zone: GMT+01:00
- Debug mode active: no
- Last backup: 26. Apr., 22:11
- Routing mode: Walk
- Live map mode: true
- OSM multi-threading: false / threads: 1
- Map: Germany (Offline)
  - Id: cgeo.geocaching.maps.mapsforge.MapsforgeMapProvider$OfflineMapSource:3438-3261:CGeo/maps/germany.map
  - Atts: Map data (c) OpenStreetMap contributors
  - Theme: Elevate.zip:Elevate.xml

Filters:
-------
- Hide waypoints: original visited
- LIVE: _Tour* ([_Tour:inconclusive=false:advanced=true]AND(status:exclude_disabled:exclude_archived))
- OFFLINE: Keine ([:inconclusive=false:advanced=false]AND(type;difficulty_terrain:d=-:d=-:t=-:t=-;status))

Stored Filters (#15):
- _Standard: [_Standard:inconclusive=false:advanced=false]AND(status:found_no:exclude_archived))
- _Test: [_Test:inconclusive=false:advanced=true]AND(difficulty_terrain:d=1.5:d=4.0:t=-:t=3.0))
- _toSolve: [_toSolve:inconclusive=false:advanced=true]AND(type:MYSTERY;status:solved_mystery_no))
- _Tour: [_Tour:inconclusive=false:advanced=false]AND(status:found_no:has_offline_found_log_no:solved_mystery_yes:exclude_disabled:exclude_archived))
- _unsaved: [_unsaved:inconclusive=false:advanced=false]AND(status:found_no:exclude_archived;stored_since:-10:-10))
- Events: [Events:inconclusive=false:advanced=false]AND(hidden:2021-09-21:-:false:-1:-1;type:EVENT))
- hasTrackable: [hasTrackable:inconclusive=false:advanced=true]AND(status:has_trackable_yes))
- LabStation: [LabStation:inconclusive=false:advanced=false]AND(note::is_present;type:USER_DEFINED:ADVLAB;status))
- Offlinelog: [Offlinelog:inconclusive=false:advanced=true]AND(status:has_offline_found_log_yes))
- Steffen: [Steffen:inconclusive=false:advanced=true]AND(status:found_no:has_offline_found_log_no;stored_list:122:31))
- toLog_Bianca: [toLog_Bianca:inconclusive=false:advanced=false]AND(stored_list:86;status:found_yes;log_entry:biansch :contains:inverse=true:logtext=:logtext=contains))
- toLog_Carolin: [toLog_Carolin:inconclusive=false:advanced=false]AND(stored_list:86;status:found_yes;log_entry:carosch:contains:inverse=true:logtext=:logtext=contains))
- Umlaute: [Umlaute:inconclusive=false:advanced=true]OR(note:+29:contains;note:+28:contains;note:+27:contains))
- winter: [winter:inconclusive=false:advanced=true]AND(attributes:winter_no:inverse=true:attributesources=3))
- WithNotes: [WithNotes:inconclusive=false:advanced=true]AND(note::is_present))

Services:
-------
- Geocaching sites enabled:
   geocaching.com: Logged in (Anmeldung OK) / PREMIUM
   Geocaching.com Adventure Lab
   opencaching.de: Logged in (Anmeldung OK)
- Geocaching.com date format: dd MMM yy
- Geocaching.com website language: en-US
- Last login error on geocaching.com: communication error (2022-11-15 15-58)
- Last successful login on geocaching.com: 2022-11-18 13-15
- Routing: internal / BRouter installed: false
- Installed c:geo plugins:  none

Permissions & paths:
-------
- Fine location permission: granted
- Write external storage permission: granted
- System internal c:geo dir: /data/user/0/cgeo.geocaching (68,0 GB free) v3 internal isDir(8 entries)
- Legacy User storage c:geo dir: /data/user/0/cgeo.geocaching (68,0 GB free) v3 internal isDir(8 entries)
- Geocache data: /storage/emulated/0/Android/data/cgeo.geocaching/files/GeocacheData (68,0 GB free) v3 external non-removable isDir(1748 entries)
- Internal theme sync (is turned off): /data/user/0/cgeo.geocaching/MapThemeData (68,0 GB free) v3 internal isDir(0 entries)
- Public Folders: #12
  - BASE: SD-Karte/CGeo (User-Defined)[SD-Karte/CGeo[DOCUMENT#0:p-content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo::]] (Uri: content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo/document/3438-3261%3ACGeo, Av:true, files:>=21, dirs:>=9, totalFileSize:>=6,7 GB, free space: 100,1 GB, files on device: 0)
  - OFFLINE_MAPS: SD-Karte/CGeo/maps (Default)[SD-Karte/CGeo/maps[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo::/maps]] (Uri: content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo/document/3438-3261%3ACGeo%2Fmaps, Av:true, files:10, dirs:2, totalFileSize:6,7 GB, free space: 100,1 GB, files on device: 0)
  - OFFLINE_MAP_THEMES: SD-Karte/CGeo/maps/_themes (Default)[SD-Karte/CGeo/maps/_themes[PERSISTABLE_FOLDER(OFFLINE_MAPS)#1:p-content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo::/maps/_themes]] (Uri: content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo/document/3438-3261%3ACGeo%2Fmaps%2F_themes, Av:true, files:2, dirs:0, totalFileSize:468,1 KB, free space: 100,1 GB, files on device: 0)
  - OFFLINE_MAP_SHADING: SD-Karte/CGeo/maps/_hgt (Default)[SD-Karte/CGeo/maps/_hgt[PERSISTABLE_FOLDER(OFFLINE_MAPS)#1:p-content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo::/maps/_hgt]] (Uri: content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo/document/3438-3261%3ACGeo%2Fmaps%2F_hgt, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 100,1 GB, files on device: 0)
  - LOGFILES: SD-Karte/CGeo/logfiles (Default)[SD-Karte/CGeo/logfiles[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo::/logfiles]] (Uri: content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo/document/3438-3261%3ACGeo%2Flogfiles, Av:true, files:1, dirs:0, totalFileSize:72,7 KB, free space: 100,1 GB, files on device: 0)
  - GPX: SD-Karte/CGeo/gpx (Default)[SD-Karte/CGeo/gpx[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo::/gpx]] (Uri: content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo/document/3438-3261%3ACGeo%2Fgpx, Av:true, files:10, dirs:0, totalFileSize:76,4 MB, free space: 100,1 GB, files on device: 0)
  - BACKUP: SD-Karte/CGeo/backup (Default)[SD-Karte/CGeo/backup[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo::/backup]] (Uri: content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo/document/3438-3261%3ACGeo%2Fbackup, Av:true, files:>=10, dirs:>=10, totalFileSize:>=253,8 MB, free space: 100,1 GB, files on device: 0)
  - FIELD_NOTES: SD-Karte/CGeo/field-notes (Default)[SD-Karte/CGeo/field-notes[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo::/field-notes]] (Uri: content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo/document/3438-3261%3ACGeo%2Ffield-notes, Av:true, files:4, dirs:0, totalFileSize:8,4 KB, free space: 100,1 GB, files on device: 0)
  - SPOILER_IMAGES: SD-Karte/CGeo/GeocachePhotos (Default)[SD-Karte/CGeo/GeocachePhotos[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo::/GeocachePhotos]] (Uri: content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo/document/3438-3261%3ACGeo%2FGeocachePhotos, Av:true, files:>=2, dirs:>=37, totalFileSize:>=5,2 MB, free space: 100,1 GB, files on device: 0)
  - ROUTING_BASE: SD-Karte/CGeo/routing (Default)[SD-Karte/CGeo/routing[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo::/routing]] (Uri: content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo/document/3438-3261%3ACGeo%2Frouting, Av:true, files:9, dirs:1, totalFileSize:340,3 MB, free space: 100,1 GB, files on device: 0)
  - ROUTING_TILES: SD-Karte/CGeo/routing/segments4 (Default)[SD-Karte/CGeo/routing/segments4[PERSISTABLE_FOLDER(ROUTING_BASE)#1:p-content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo::/routing/segments4]] (Uri: content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo/document/3438-3261%3ACGeo%2Frouting%2Fsegments4, Av:true, files:2, dirs:0, totalFileSize:340,2 MB, free space: 100,1 GB, files on device: 0)
  - TEST_FOLDER: [Legacy]/data/user/0/cgeo.geocaching/files/unittest (Default)[/data/user/0/cgeo.geocaching/files/unittest[FILE#1:p-file:///data/user/0/cgeo.geocaching/files::/unittest]] (Uri: file:///data/user/0/cgeo.geocaching/files/unittest, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 68,0 GB, files on device: -1)
- Map render theme path: Elevate.zip:Elevate.xml
- PersistedDocumentUris: #1
- UNUSED_TRACK: null
- Persisted Uri Permissions: #1
  - content://com.android.externalstorage.documents/tree/3438-3261%3ACGeo (1. Apr., 19:00):RW
- Database: /data/user/0/cgeo.geocaching/databases/data (v99, Size:99,2 MB) on system internal storage

--- End of system information ---
```

### Additional Information

[logcat-2022-11-18-13-15-21-2.txt](https://github.com/cgeo/cgeo/files/10041188/logcat-2022-11-18-13-15-21-2.txt)

**Comment by eddiemuc** [comment]
confirmed

**Comment by eddiemuc** [comment]
Fixed with PR #13692
